assets/js/shortcake-bakery.js

```diff
@@ -40,6 +40,16 @@
 
 	$(document).ready(function(){
 		if ( $('.shortcake-bakery-responsive').length ) {
+			$('.shortcake-bakery-responsive').each(function(){
+				var el = $(this);
+				// Persist the original proportion when true height and width aren't declared
+				if ( el.attr('height') && el.attr('width')
+					&& ! el.attr('data-true-width') && ! el.attr('data-true-height')
+					&& -1 === el.attr('height').indexOf('%') && -1 === el.attr('width').indexOf('%') ) {
+					el.attr('data-true-height', el.attr('height'));
+					el.attr('data-true-width', el.attr('width'));
```

Comment:
Since you don't need to write to markup here, why not just set the data attributes using `el.data('true-width', el.attr('width'));`? The values are being referenced by `el.data` in `responsiveElements()`, so it seems more consistant to do the same here.